pkg/runner/runner.go

```diff
@@ -162,14 +162,15 @@ func (r *Runner) processSubdomains() {
 // runMassdns runs the massdns tool on the list of inputs
 func (r *Runner) runMassdns(inputFile string) {
 	massdns, err := massdns.New(massdns.Config{
-		Domain:        r.options.Domain,
-		Retries:       r.options.Retries,
-		MassdnsPath:   r.options.MassdnsPath,
-		Threads:       r.options.Threads,
-		InputFile:     inputFile,
-		ResolversFile: r.options.ResolversFile,
-		TempDir:       r.tempDir,
-		OutputFile:    r.options.Output,
+		Domain:           r.options.Domain,
+		Retries:          r.options.Retries,
+		MassdnsPath:      r.options.MassdnsPath,
+		Threads:          r.options.Threads,
+		WildcardsThreads: r.options.WildcardThreads,
```

Comment:
Modify the readme to specify what is wildcard threads and how to use it.

Reply:
done